modules/load_balancer/lb_data.c

```diff
@@ -778,7 +779,16 @@ int lb_route(struct sip_msg *req, int group, struct lb_res_str_list *rl,
 	}
 	/* choose one destination among selected */
 	if( dsts_size_cur > 0 ) {
-		if( (dsts_size_cur > 1) && (flags & LB_FLAGS_RANDOM) ) {
+		if((dsts_size_cur > 1) && (flags & LB_FLAGS_QUEUE)){
+			//Find lowest queue number in virtual queue
+			q_lowest = data->dst_no;
+			for( i=0 ; i<dsts_size_cur; i++){
+				if (dsts[i]->queue_loc < q_lowest){
```

Comment:
Harmless, but for code consistency, this should be `dsts_cur[i]->queue_loc`, as below.